srcs/runners_manager/vm_creation/gcloud/GcloudManager.py

```diff
@@ -73,6 +74,18 @@ def configure_instance(
         )
         disk_size_gb = runner.vm_type.config["disk_size_gb"]
         disk_type = f"projects/{self.project_id}/zones/{self.zone}/diskTypes/pd-ssd"
+
+        automatic_restart = True
+        provisioning_model = "STANDARD"
+        instance_termination_action = "DEFAULT"
+        preemptible = False
+        if "spot" in runner.vm_type.config:
+            preemptible = bool(runner.vm_type.config["spot"])
+            if preemptible:
+                provisioning_model = "SPOT"
+                automatic_restart = False
+                instance_termination_action = "DELETE"
```

Comment:
Considering marshmallow defaults are not applied:

Option 1
```suggestion
        preemptible = bool(runner.vm_type.config.get("spot", False))
        if preemptible:
            provisioning_model = "SPOT"
            automatic_restart = False
            instance_termination_action = "DELETE"
```

Reply:
Done!